crates/rpc-types-mev/src/mev_calls.rs

```diff
@@ -144,14 +144,12 @@ pub struct SimBundleResponse {
     #[serde(with = "alloy_serde::quantity")]
     pub state_block: u64,
     /// The gas price of the simulated block.
-    #[serde(with = "alloy_serde::quantity")]
-    pub mev_gas_price: u64,
+    pub mev_gas_price: U256,
     /// The profit of the simulated block.
-    #[serde(with = "alloy_serde::quantity")]
-    pub profit: u64,
+    #[serde(rename = "profit")]
+    pub total_profit: U256,
```

Comment:
mev_share docs mentions "profit" in the rpc response(https://docs.flashbots.net/flashbots-auction/advanced/rpc-endpoint#mev_simbundle), and the golang implementation also renames TotalProfit to profit: https://github.com/flashbots/builder/blob/df9c765067d57ab4b2d0ad39dbb156cbe4965778/internal/ethapi/sbundle_api.go#L285

Maybe we can just call it profit because there doesnt seem to be any major difference semantically between the two and the bundle spec requires "profit"